package.json

```diff
@@ -47,6 +47,6 @@
     "metals": "tail -F .metals/metals.log | stdbuf -oL cut -c 21- | rg -v '(notification for request|handleCancellation)'",
     "serverlog": "pnpm journal & pnpm metals",
     "multilog": "pnpm serverlog & ui/build -r",
-    "minilog": "pnpm journal | grep -E -v fishnet & ui/build -dr"
+    "minilog": "pnpm journal | grep -E -v fishnet & ui/build -cdr"
```

Comment:
?

Reply:
-cdr is just like -dr except it runs a _clean_ build after every detected package.json change. So it removes intermediate compile state (tsconfig.tsbuildinfo) that could can trick the build into succeeding when it should actually fail.

Reply:
Yes, what I meant is, I don't see how it relates to moderating ublogs

Reply:
It doesn't, you can revert it. But AFAIK noone uses that script but me.